Html and Css files added

GhostEsso opened the pull request:
- I updated the readme.md file
- I configured the linters
- I corrected some fixes on html and css files

GhostEsso committed:
add index.html

GhostEsso committed:
add css file

GhostEsso committed:
linters corrections

GhostEsso committed:
linters corrections

GhostEsso committed:
linters corrections

GhostEsso committed:
linters corrections

GhostEsso reviewed (commented):
Hi @GhostEsso Please, you have to document your repository in a professional way. 
![image](https://user-images.githubusercontent.com/114746721/220084713-caa3e825-ce9f-4a6c-b687-6375d9c8e917.png)

@GhostEsso  I think it would be better using this example
![Screenshot from 2023-02-20 11-03-36](https://user-images.githubusercontent.com/114746721/220087955-4cb2753f-64fe-4bc0-8de4-fdc1f5898495.png)

GhostEsso reviewed (commented):
@GhostEsso You have to add  a descriptive README file to your project. 
![image](https://user-images.githubusercontent.com/114746721/220086162-ca398e21-ec97-4aaa-9ac8-929d0b6e3ff2.png)


You can use this template [https://github.com/microverseinc/readme-template](url)

GhostEsso committed:
Update Readme.md

GhostEsso committed:
Update README.md

AlphaNtihinduka reviewed (changes requested):
## Hi @GhostEsso 🙋‍♂️

Good job working on your review, you have done to the best expectation and incredibly well. 👏  
However, there are some issues mostly on the\ use on inline comments and communication.

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

![close](https://user-images.githubusercontent.com/99394374/210771148-619a4265-33b2-4e68-8901-6bb52693d6de.gif)

## Required changes ♻️ 
- [ ] You have done well to request the student to provide the description 👏 . However, you didn't provide detailed explaination `why` detailed  PR description title and body are important. Kindly ensure to explain so tha the student can clearly understand you comment 👍. Therefore, i kindly request that you provide detailed explaination on the changes you requested on the description.
- [ ] I suggest that you try as much as possible to use the inline comments as explained in this [documentation](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) of how to use the github features. This will help the student to know exactly where the changes should be applied ☺️ . I therefore request that you make a re-review on the Readme.

## Optional suggestions
- [ ] [OPTIONAL] Kindly make sure that you atleast provide a comment to the technical part of the PR. This will give you a better practice and become a code reviewer gulu 🥰 



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

wayungi reviewed (changes requested):
Hi @GhostEsso 🤝 , 
![image](https://user-images.githubusercontent.com/99656687/220092851-74c5dee6-6821-4304-b4e4-011430fce3d4.png)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. 😃 


 ## Highlights
- [ ] Good use of GitHub features✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Please use [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) to do your reviews.
- [ ] Please give at least 2 reviews. Remeber to be polite, and use inline comments and offer explanations why a change is required.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

GhostEsso commented on README.md:
```diff
@@ -1 +1,236 @@
+- [📝 License](#license)
+
+
+# 📖 Portofolio: set up and mobile
```
Hi @GhostEsso I think it would be nice if your project name matched the name given in your readme file.

![image](https://user-images.githubusercontent.com/114746721/220098409-6d6cf57a-b58a-49b4-bc7d-928dd33eed0c.png)

GhostEsso committed:
Update Readme .md file

I corrected the title I gave for my project in the readme file to better match the project

GhostEsso commented on index.html:
```diff
@@ -0,0 +1,138 @@
+
```
@GhostEsso it would be wise for you to remove this empty space at the beginning of your html code


![Screenshot from 2023-02-20 11-59-23](https://user-images.githubusercontent.com/114746721/220101437-b7b2f21b-d530-4cf0-b051-370174f5ff1d.png)

GhostEsso commented on style.css:
```diff
@@ -0,0 +1,60 @@
+  cursor: pointer;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  width: calc(100% / 3);
+  margin: 10px;
+  border: 1px solid #000;
+  border-radius: 3px;
+}
```
@GhostEsso Please, you can increase the size of your border-radius to 5px to make it more consistent

GhostEsso commented on style.css:
```diff
@@ -0,0 +1,60 @@
+  text-transform: uppercase;
+  font-weight: 700;
+}
+
+.product > h2 {
+  margin-top: 15px;
+}
+
```
@GhostEsso Please, you can decrease your margin-top to 10 px, it  would be better

GhostEsso reviewed (commented):
# Changes required :recycle: 

Please check my comments below and make the all required changes

Submit for a re-view when you're done

Thanks :grin:

ridwanediallo reviewed (approved):
# STATUS: APPROVED 🟢
## Hi @GhostEsso , 
![Congratulations!GIF](https://user-images.githubusercontent.com/55918904/191301390-43d62dfa-bfdc-49c5-943b-4d72f4ea5bb8.gif)

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- Your comments are polite. ✔
- Good use of the inline comments. ✔
- You explained why you requested the changes. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
